src/db/tests/db.rs

```diff
@@ -471,6 +472,8 @@ async fn delete_collection() -> Result<()> {
             collection: coll.to_owned(),
         })
         .await?;
+    // ### returns a timestamp of 0 because the collection is gone.
```

Comment:
is this comment outdated? this test looks like it would fail if it's still the case